inc/ocf_core.h

```diff
@@ -29,7 +29,7 @@ struct ocf_core_info {
 
 		/** Number of blocks left to flush in ongoing flush operation */
 		uint32_t dirty;
-	};
+	}flush_operation;
```

Comment:
This unnecessarily breaks backwards compatibility. Why not just flatten this structure instead?